src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -371,22 +372,22 @@ public void setBrowser(GitRepositoryBrowser browser) {
             ;
 
     private static final Pattern[] URL_PATTERNS = {
-        /* URL style - like https://github.com/jenkinsci/git-plugin */
-        Pattern.compile(
-        "(?:\\w+://)" // protocol (scheme)
-        + "(?:.+@)?" // optional username/password
-        + HOSTNAME_MATCH
-        + "(?:[:][\\d]+)?" // optional port number (only honored by git for ssh:// scheme)
-        + "/" // separator between hostname and repository path - '/'
-        + REPOSITORY_PATH_MATCH
-        ),
-        /* Alternate ssh style - like git@github.com:jenkinsci/git-plugin */
-        Pattern.compile(
-        "(?:git@)" // required username (only optional if local username is 'git')
-        + HOSTNAME_MATCH
-        + ":" // separator between hostname and repository path - ':'
-        + REPOSITORY_PATH_MATCH
-        )
```

Comment:
Please follow the instructions in the [contributing guide](https://github.com/jenkinsci/git-plugin/blob/master/CONTRIBUTING.adoc) that say:

>  Please don’t perform wholesale reformatting of a file without discussing with the current maintainers.

There are many pending pull requests open in this plugin.  When one pull request performs a reformat, it risks creating conflicts in other pull requests.  Please remove all reformatting changes.